.github/scripts/run_ci.sh

```
@@ -122,6 +122,15 @@ if [ $testResult -ne 0 ]; then
 	exit $testResult
 fi
 
+cd packages/app-desktop && yarn install-electron
+testResult=$?
+if [ $testResult -ne 0 ]; then
+	echo "Failed to install Electron."
+	exit $testResult
+fi
+
+cd "$ROOT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)